docs/src/operations/guides/validator-start.md

```diff
@@ -216,7 +216,7 @@ Or to see in finer detail:
 solana balance --lamports
 ```
 
-Read more about the [difference between SOL and lamports here](https://solana.com/docs/intro#what-are-sols).
+Read more about the [difference between SOL and lamports here](https://docs.solana.com/developing/programming-model/accounts#lamports).
```

Comment:
Agreed that the old link is bad, but I think we can do better than the link you provided too. It looks like there are some relevant sections in these spots:
https://solana.com/docs/references/terminology#lamport
https://solana.com/docs/references/terminology#sol